go-controller/pkg/ovn/multicast_test.go

```diff
@@ -441,10 +441,15 @@ var _ = ginkgo.Describe("OVN Multicast with IP Address Family", func() {
 				err := fakeOvn.controller.disableMulticast()
 				gomega.Expect(err).NotTo(gomega.HaveOccurred())
 				// check acls are deleted when multicast is disabled
-				dereferenceAllACLs(initialData)
```

Comment:
I think dereferenceAllACLs and dereferenceACLsFromLastPG are not used anymore and may be deleted